system_reports/MACOS/M1/osx-xcode-13.4.x.log

```diff
@@ -2973,8 +2971,8 @@ android-32
 
  * SDK packages:
 * Command line Tools version: 6.0
-* Platform Tools version: 33.0.1
-* Emulator version: 31.2.10
```

Comment:
Can we keep 31.2.10 on Intel? On M1 it does not matter.

Reply:
we son't hardcode the version just run `sdkmanager install emulator` so if it's like this then so be it